pkg/azuredisk/controllerserver.go

```diff
@@ -98,7 +98,13 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	capacityBytes := req.GetCapacityRange().GetRequiredBytes()
 	volSizeBytes := int64(capacityBytes)
 	requestGiB := int(volumehelper.RoundUpGiB(volSizeBytes))
+
+	if diskParams.PerformancePlus != nil && *diskParams.PerformancePlus {
```

Comment:
`if ... && requestGiB <  consts.PerformancePlusMinimumDiskSizeGiB`

Reply:
Ah, sure, fixed.

Reply:
still not fixed?

Reply:
sorry, now it should be ok